Cargo.toml

```diff
@@ -24,6 +24,7 @@ rustc-serialize = "^0.3.15"
 num = "^0.1.25"
 lazy_static = "^0.1.14"
 quick-error = "^0.1.3"
+pulldown-cmark={ git="https://github.com/google/pulldown-cmark.git"}
```

Comment:
Yes, the project is not very well documented so I had to pull the rendering code from the main function in the master branch. When I tried to compile it seemed clear that there were some API changes.

Is there any way to fix the specific commit that you want to use of that repo? At least then it won't break as soon as the other author changes the API.


```
src/helpers/helper_markdown.rs:8:28: 8:35 error: unresolved import `self::pulldown_cmark::Options`. There is no `Options` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                            ^~~~~~~
src/helpers/helper_markdown.rs:8:28: 8:35 help: run `rustc --explain E0432` to see a detailed explanation
src/helpers/helper_markdown.rs:8:37: 8:57 error: unresolved import `self::pulldown_cmark::OPTION_ENABLE_TABLES`. There is no `OPTION_ENABLE_TABLES` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                                     ^~~~~~~~~~~~~~~~~~~~
src/helpers/helper_markdown.rs:8:37: 8:57 help: run `rustc --explain E0432` to see a detailed explanation
src/helpers/helper_markdown.rs:8:59: 8:82 error: unresolved import `self::pulldown_cmark::OPTION_ENABLE_FOOTNOTES`. There is no `OPTION_ENABLE_FOOTNOTES` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                                                           ^~~~~~~~~~~~~~~~~~~~~~~
src/helpers/helper_markdown.rs:8:59: 8:82 help: run `rustc --explain E0432` to see a detailed explanation
error: aborting due to 3 previous errors
Build failed, waiting for other jobs to finish...
src/helpers/helper_markdown.rs:8:28: 8:35 error: unresolved import `self::pulldown_cmark::Options`. There is no `Options` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                            ^~~~~~~
src/helpers/helper_markdown.rs:8:28: 8:35 help: run `rustc --explain E0432` to see a detailed explanation
src/helpers/helper_markdown.rs:8:37: 8:57 error: unresolved import `self::pulldown_cmark::OPTION_ENABLE_TABLES`. There is no `OPTION_ENABLE_TABLES` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                                     ^~~~~~~~~~~~~~~~~~~~
src/helpers/helper_markdown.rs:8:37: 8:57 help: run `rustc --explain E0432` to see a detailed explanation
src/helpers/helper_markdown.rs:8:59: 8:82 error: unresolved import `self::pulldown_cmark::OPTION_ENABLE_FOOTNOTES`. There is no `OPTION_ENABLE_FOOTNOTES` in `helpers::helper_markdown::pulldown_cmark` [E0432]
src/helpers/helper_markdown.rs:8 use self::pulldown_cmark::{Options, OPTION_ENABLE_TABLES, OPTION_ENABLE_FOOTNOTES};
                                                                                           ^~~~~~~~~~~~~~~~~~~~~~~
src/helpers/helper_markdown.rs:8:59: 8:82 help: run `rustc --explain E0432` to see a detailed explanation
error: aborting due to 3 previous errors
Could not compile `handlebars`.
```